src/components/MovieCard2.vue

```diff
@@ -1,18 +1,17 @@
 <template>
```

Comment:
When I use Movie Card with props I can't style it the way I want, but maybe I'm doing something wrong :) I can show it to you on slack or zoom because it's hard to explain :)

Reply:
we can talk on zoom or slack if you need it, may be I can help with that :)